### ISSUE TITLE
basset的drop_rate似乎并没有用到，这样的话似乎这边不应该用ReduceLROnPlateau

### ISSUE THREAD

**Issue by sybwjdnr** [gao-lab/VCNN#8]
https://github.com/davek44/Basset/blob/8b672320c348c6610ad79015457e0afa1c18e456/src/basset_train.lua

cmd:option('-drop_rate', false, 'Decrease the learning_rate when training loss stalls')

if opt.drop_rate and train_loss_last ~= nil and (train_loss_last - train_loss)/train_loss_last < .0001 then
        convnet:drop_rate()
        io.write(", rate drop")
    end
